packages/renderer/src/lib/preferences/item-formats/StringItem.spec.ts

```diff
@@ -0,0 +1,25 @@
+import '@testing-library/jest-dom/vitest';
+import { test, expect, vi, beforeAll } from 'vitest';
+import { render, screen } from '@testing-library/svelte';
+import type { IConfigurationPropertyRecordedSchema } from '../../../../../main/src/plugin/configuration-registry';
+import StringItem from '/@/lib/preferences/item-formats/StringItem.svelte';
+
+beforeAll(() => {
+  (window as any).getConfigurationValue = vi.fn();
+});
+
+test('Ensure HTMLInputElement', async () => {
```

Comment:
might miss placeholder and readonly properties test/check

Reply:
done